auxx/src/Command/Tx.hs

```diff
@@ -212,36 +214,58 @@ newtype AuxxException = AuxxException Text
 instance Exception AuxxException
 
 send
-    :: forall m. MonadAuxxMode m
+    :: forall m
+     . MonadAuxxMode m
     => ProtocolMagic
+    -> SlotCount
     -> Diffusion m
     -> Int
     -> NonEmpty TxOut
     -> m ()
-send pm diffusion idx outputs = do
+send pm epochSlots diffusion idx outputs = do
     skey <- takeSecret
     let curPk = encToPublic skey
-    let plainAddresses = map (flip makePubKeyAddress curPk . IsBootstrapEraAddr) [False, True]
-    let (hdAddresses, hdSecrets) = unzip $ map
-            (\ibea -> fromMaybe (error "send: pass mismatch") $
-                    deriveFirstHDAddress (IsBootstrapEraAddr ibea) emptyPassphrase skey) [False, True]
+    let plainAddresses = map
+            (flip makePubKeyAddress curPk . IsBootstrapEraAddr)
+            [False, True]
+    let
+        (hdAddresses, hdSecrets) = unzip $ map
+            (\ibea ->
+                fromMaybe (error "send: pass mismatch") $ deriveFirstHDAddress
+                    (IsBootstrapEraAddr ibea)
+                    emptyPassphrase
+                    skey
+            )
+            [False, True]
     let allAddresses = hdAddresses ++ plainAddresses
-    let allSecrets = hdSecrets ++ [skey, skey]
-    etx <- withSafeSigners allSecrets (pure emptyPassphrase) $ \signers -> runExceptT @AuxxException $ do
-        let addrSig = HM.fromList $ zip allAddresses signers
-        let getSigner addr = HM.lookup addr addrSig
-        -- BE CAREFUL: We create remain address using our pk, wallet doesn't show such addresses
-        (txAux,_) <- lift $ prepareMTx pm getSigner mempty def (NE.fromList allAddresses) (map TxOutAux outputs) curPk
-        txAux <$ (ExceptT $ try $ submitTxRaw diffusion txAux)
+    let allSecrets   = hdSecrets ++ [skey, skey]
+    etx <- withSafeSigners allSecrets (pure emptyPassphrase) $ \signers ->
+        runExceptT @AuxxException $ do
+            let addrSig = HM.fromList $ zip allAddresses signers
+            let getSigner addr = HM.lookup addr addrSig
+            -- BE CAREFUL: We create remain address using our pk, wallet doesn't
+            -- show such addresses
+            (txAux, _) <- lift $ prepareMTx pm
+                                            epochSlots
+                                            getSigner
+                                            mempty
+                                            def
+                                            (NE.fromList allAddresses)
+                                            (map TxOutAux outputs)
+                                            curPk
```

Comment:
The original line was obvious too long because we can't see the end of the line in the Github diff.

I would probably have formatted that as:
```
            (txAux, _) <- lift $ prepareMTx pm epochSlots getSigner mempty def
                                   NE.fromList allAddresses) (map TxOutAux outputs) curPk
```
which is still under 100 chars wide.